others/RESOURCES.md

```diff
@@ -11,3 +11,10 @@ Anything useful or informative that can be found on the internet, including arti
 * [mCoding](https://www.youtube.com/@mCoding) - Python
 * [Luke Smith](https://www.youtube.com/@LukeSmithxyz) - Linux, Shells, Life
 * [Hussein Nassar](https://www.youtube.com/@hnasr) - Backend System Design
+* [Honeypot](https://www.youtube.com/@Honeypotio) - Interesting documentaries about different technologies
+* [Fireship.io](https://www.youtube.com/@Fireship) - JS, TS, Code Reports, 100s
+* [OWASP](https://owasp.org/) - Cybersecurity
```

Comment:
This is not a YouTube channel. Create a new subsection for resourceful websites.

Reply:
Done